amy/templates/base.html

```diff
@@ -33,9 +33,19 @@
   <body>
 
     {% if SITE_BANNER_STYLE == "local" %}
-    <section class="banner banner-local">This is a local version of AMY.</section>
+    <section class="banner banner-local">
+      This is a local version of AMY.
+      {% if user.is_admin %}
+      You have {{ FEATURE_FLAGS_ENABLED|length }} feature flag{{ FEATURE_FLAGS_ENABLED|length|pluralize }} <a href="{% url 'feature_flags' %}">enabled</a>.
```

Comment:
@elichad I'm going to show this line in the banner unconditionally for the users. The link will require logging in, so in future if you want to allow unauthorized users to participate in testing, there may be additional work required to enable that view for them, or another approach to enabling the feature flag will have to be taken.

Reply:
> For membership codes, I was thinking to just omit the anonymous condition from the flag, because that should then allow the flag to be set with a URL parameter even without access to the view (right?).

Yes, it would work like that.